<?xml version="1.0"?>
<launch>

    <node pkg="arena_camera_ros2" exec="arena_camera_ros2_node" name="arena_camera_node" output="screen">
        <param name="serial" value="221202871" />
        <param name="gain" value="10.0" />
        <param name="exposure_time" value="2000.0" />
        <param name="gamma" value="0.5" />
        <param name="qos_reliability" value="reliable" /> <!-- "best_effort" or "reliable"-->
        <param name="camera_info_file" value="$(find-pkg-share arena_camera_ros2)/camera_intrinsics/camera_info.yaml" />

        <remap from="image_raw" to="test_cam/image_raw" />
        <remap from="camera_info" to="test_cam/camera_info" />
    </node>

</launch>